server.js

```diff
@@ -0,0 +1,75 @@
+import mongoose from "mongoose";
+import dotenv from "dotenv";
+import express from "express"
+import bodyParser from "body-parser"
+import cors from "cors"
+import validator from 'validator';
+
+dotenv.config({ path: "./config.env" });
+
+const connectionString = process.env.MONGO_URL
+const database = await mongoose.connect(connectionString);
+const app = express();
+
+console.log(`Connected to ${connectionString}`);
+
+const userSchema = mongoose.Schema({ username: String, email: String, password: String, userUID: String, isInstructor: { type: Boolean, default: false } });
+const userModel = database.model("users", userSchema);
+const users = await userModel.find();
+
+
+const urlencodedParser = bodyParser.urlencoded({ extended: false });
+app.use(bodyParser.json(), urlencodedParser)
+app.use(cors())
+
+
+app.listen(process.env.PORT, () => {
+    console.log(`App is Listening on PORT ${process.env.PORT}`)
+})
+
+app.get("/", async (req, res) => {
+    const users2 = await userModel.find();
```

Comment:
The application will crash here if MongoDB can't be reached.  The OpenAPI specification doesn't state how to handle this situation, so I'll update it to include a Status 503 response & message.  This function call should thus be wrapped in a `try-catch` block and this situation handled appropriately.